### PR TITLE
feat: Add DataTables' generic fragment

### PR DESCRIPTION
**Pull request by tarun0**
Fix: #466 
Currently added in Client Loan Application only. Adding to the others as well in the meantime.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by therajanmaurya** [comment]
@tarun0 Good Work.

**Comment by tarun0** [comment]
Thanks @therajanmaurya  :)

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/datatablelistfragment/DataTableListPresenter.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                .subscribe(new Subscriber<Loans>() {
+                    @Override
+                    public void onCompleted() {
+                        getMvpView().showProgressbar(false);
+                    }
+
+                    @Override
+                    public void onError(Throwable e) {
+                        getMvpView().showProgressbar(false);
+                        getMvpView().showMessage(R.string.generic_failure_message, "fail");
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/datatablelistfragment/DataTableListPresenter.java]: Remove the hard coded string

File: mifosng-android/src/main/java/com/mifos/objects/datatablepayload/DataTable.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import android.os.Parcelable;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.HashMap;
+
+/**
+ * Created by Tarun on 1/28/2017.
+ */
+public class DataTable implements Parcelable {
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/objects/datatablepayload/DataTable.java]: Rename with DataTablePayload and put in the object/noncore/

**Pull request merged**